.eslintrc.js

```diff
@@ -3,43 +3,58 @@ module.exports = {
     es2021: true,
     node: true
   },
-  extends: 'standard-with-typescript',
+  extends: [
+    'bloq',
+    'prettier',
+    'standard-with-typescript'
```

Comment:
´prettier´ should always be the last one to properly apply formattings.

What rules is `standard-with-typescript` adding? Any worth adding to `eslint-config-bloq` instead? cc/ @gndelia

Reply:
I removed `standard-with-typescript` and got the following weird linting errors:

![Captura de Tela 2025-03-21 às 13 02 18](https://github.com/user-attachments/assets/14ddda52-db65-462a-b407-f5a2afb9fd3d)

*seems like it does not recognize some Typescript reserved keyword and some `@typescript-eslint` rules

Reply:
Updated to let `prettier` be the last in the extends array